src/components/Citizen.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react";
+import CitizenData from "./CitizenData";
+import { GetBPDCitizenT } from "../generated/definitions/requestTypes";
+import { BPDCitizen } from "../generated/definitions/BPDCitizen"
+import { fromEither, fromLeft, tryCatch } from "fp-ts/lib/TaskEither";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { TypeofApiResponse } from "italia-ts-commons/lib/requests";
+import {simpleClient} from "../helpers/client";
+import { toError } from "fp-ts/lib/Either";
+import { useTranslation } from 'react-i18next';
+
+
+function Citizen(props) {
+    const [resultData, setResultdata] = useState<BPDCitizen | undefined>(undefined);
+    const [resultErr, setResulterr] = useState<string>("");
+
+    const { t, i18n } = useTranslation();
+
+    useEffect(() => {
+      if (props.location.state) {
+        const citizenid = props.location.state.citizenid;
+        // useful to have value also in case of page-refresh
+        window.sessionStorage.setItem('citizenid',props.location.state.citizenid);
+      }
+
+      // TaskEither
+      tryCatch(() => simpleClient.GetBPDCitizen({
+        'x-citizen-id': window.sessionStorage.getItem('citizenid') || "",
```

Comment:
I suggest to have a method getCitizenId() => window.sessionStorage.getItem('citizenid')